### DISCUSSION TITLE
Collapsible content on blocks

### DISCUSSION THREAD

**Discussion by ijpatricio** [Z3d0X/filament-fabricator#48] [Ideas]
Hello @Z3d0X 

I have a block with a relatively big schema, especially in UI, because it has a few Curator fields.
This makes it a little bit hard to re-position. I can zoom out to 10%, and work it like that, no problem.

A more viable solution would be collapsible content, like in the curator field. 

Hopefully, the photos below help to visualize what I mean 😃 

Let me know what you think! I would like to try to do it as PR, by the way 👍 

![image](https://user-images.githubusercontent.com/26031459/221777573-afcbedc4-9b42-44a9-8382-3e8cd5d0f25d.png)

![image](https://user-images.githubusercontent.com/26031459/221776937-5344f3bb-4d6a-4301-ab74-b075a23a6500.png)

**Comment by ijpatricio** [comment]
And thank you so much for coming on the stream. It was a pleasure to have you there 💯

**Comment by Z3d0X** [comment]
You can get the collapsible behavior by placing this in a service provider.

```php
use Z3d0X\FilamentFabricator\Forms\Components\PageBuilder;
 
PageBuilder::configureUsing(function (PageBuilder $builder) {
    $builder->collapsible();
    $builder->collapsed(); // if you want all blocks to be collapsed by default
});
```

ref: https://filamentphp.com/plugins/fabricator#page-builder

**Comment by ijpatricio** [comment]
Ouch, I feel stupid now 😅 read the docs! (I have read them, but then I forgot) 😄 

That's great. Would you like some pictures to illustrate that right on the docs? I think that would help.

Additionally, regarding the feature`Parent page, I cant seem to find it anywhere, is it documented?

Not only photos, maybe I could make a series of short videos for Fabricator? Is that something you would like?
